crates/admin/src/cluster_controller/service.rs

```diff
@@ -225,18 +226,36 @@ impl ClusterControllerHandle {
     pub async fn create_partition_snapshot(
         &self,
         partition_id: PartitionId,
-    ) -> Result<Result<Snapshot, anyhow::Error>, ShutdownError> {
+        min_target_lsn: Option<Lsn>,
+        trim_log: Option<bool>,
```

Comment:
I think this can be just `bool`. then do `request.trim_log.unwrap_or_default()`

Reply:
updated, thx!